optaplanner-docs/src/main/asciidoc/MoveAndNeighborhoodSelection/MoveAndNeighborhoodSelection-chapter.adoc

```diff
@@ -259,13 +264,14 @@ Advanced configuration:
 [source,xml,options="nowrap"]
 ----
     <pillarChangeMoveSelector>
+      <subPillarType>SEQUENCE</subPillarType>
+      <subPillarSequenceComparatorClass>...</subPillarSequenceComparatorClass>
```

Comment:
Ah, it's fixed in the 2 other cases, but not on this one. As this is the first, I'd recommend to fix it here too, wdyt?

Reply:
I could have sworn I did. OK, I'm on it.